OpenSim/Analyses/JointReaction.h

```diff
@@ -72,12 +72,12 @@ OpenSim_DECLARE_CONCRETE_OBJECT(JointReaction, Analysis);
         /* Joint reference*/
         const Joint* joint;
         /* The body upon which the force is applied */
-        const PhysicalFrame* appliedOnBody;
+        const Frame* appliedOnBody;
```

Comment:
We could potentially have `OffsetFrames` of non-`PhyiscalFrames`. For example, a fixed offset to the frame computed to locate the CoP and be oriented according to the GRF direction. We don't have this use case yet, but it does indicate that a base frame could be computed/floating (opposed to "physical"). I think it is fine that `findBaseFrame()` returns a `Frame`. I don't think that is the issue here.  The point I was raising is that a `Joint` can only connect (and apply forces) to `PhysicalFrames` so it seems odd that the internal variable that represents that PhysicalFrame isn't of type `PhysicalFrame`.  I cannot think of any way that `appliedOnBody` could not be a `PhysicalFrame` so it probably is not worth verifying via `dynamic_cast`. I would leave as it since it is in a private structure only used by the analysis.